src/clj/rems/email/template.clj

```diff
@@ -29,6 +29,9 @@
    (when-not (empty? (:application/description application))
      (str ", \"" (:application/description application) "\""))))
 
+(defn- format-workflow-for-email [workflow]
+  (:name workflow))
```

Comment:
this is just a purely stylistic comment as i'm not yet familiar with most of the codebase

could this function be inlined where it's used currently? or would it be too ambiguous that way? just seeing as on row 294 is similarly `(:invitation/name invitation)`

Reply:
Yeah for this it is likely overkill. This was mimicking the one for the applications where it is more complicated. I'll simplify this. Also I'm not actually sure there is is a workflow `:name`... isn't it `:title`? :thinking:

Reply:
Yes it was supposed to be `:title` but this is now fixed.